Tella/Utils/Validator.swift

```diff
@@ -74,6 +74,24 @@ extension String {
         }
         return true
     }
+    func uwaziUsernameValidator() -> Bool {
```

Comment:
Since both `uwaziUsernameValidator` and `uwaziPasswordValidator ` are using the same logic. I think we could unify these two methods in one like this:

```
func validateWithRegex(pattern: String) -> Bool {
    guard !self.isEmpty else {
        return false
    }
    return validateRegex(value: self, pattern: pattern)
}
```

Reply:
Ya that makes scene but I think we should only use this function `validateWithRegex ` in the extension and remove all other validators from the extension and just place it into `viewmodel` where it is used so it is better for testability and good for readability.

As this branch is for the uwazi login validation I have not change the code as I explained but I have combined it into one function for now 

```Swift
func uwaziLoginValidtor() -> Bool {
        guard !self.isEmpty else {
            return false
        }
        guard validateRegex(value: self, pattern:  Regex.textRegex) else {
            return false
        }
        return true
    }
```